website/docs/Installation/macos-install.md

```diff
@@ -44,7 +44,7 @@ You will be redirected to the Podman Installer. Follow the instructions on scree
 
 ![img4](img/system-pass.png)
 
-After installation is complete, close the installer. Podman Engine has been installed and you are now ready to use Podman Desktop.
+After installation is complete, close the installation program. Podman Engine has been installed and you are now ready to use Podman Desktop.
```

Comment:
```suggestion
After the installation is complete, close the installation program. Podman Engine has been installed and you are now ready to use Podman Desktop.
```

Would this be better? I feel like we're missing the `the`.

Reply:
You are right. I was entirely focusing on Vale errors to keep the scope of the pr small.